MetaMask/metamask-extension
# Update display of switching current network

Fixes: #12691

Explanation:  

Design is updated when switching to a new network.

Manual testing steps:  
  - Use  https://chainlist.org/  to switch network in MetaMask
  - New design for network switching should be shown
  - 
![image](https://user-images.githubusercontent.com/97883527/151781554-6de4e0c9-c8c3-44f0-9e93-0ebe9b61f479.png)
